examples/grafana/nginx-mesh-top.json

```diff
@@ -350,8 +333,11 @@
       "linewidth": 1,
       "links": [],
       "nullPointMode": "null",
+      "options": {
+        "alertThreshold": true
+      },
       "percentage": false,
-      "pluginVersion": "7.1.5",
+      "pluginVersion": "7.5.3",
```

Comment:
does this work even if the version of grafana is older than 7.5.3?

Reply:
Yes, the version gets reverted back.